eduvpn/actions/add.py

```diff
@@ -7,11 +7,12 @@
 from eduvpn.metadata import Metadata
 from eduvpn.steps.instance import fetch_instance_step
 from eduvpn.steps.custom_url import custom_url
+import Gtk
```

Comment:
This should be `from gi.repository import Gtk`, travis fails.

Reply:
I found several of these import Gtk. I'll check all the files for more of these wrong imports.

Reply:
yeah, thats weird. not sure what happened there, travis seems to fail on it.

Reply:
On master it is only `eduvpn/actions/vpn_status.pyi`  that contains a `import Gtk`. This is an error, but remained unnoticed until now.